src/instruments/src/EFB/Localization/th.json

```diff
@@ -135,23 +135,23 @@
       "Unavailable": "ไม่พร้อมใช้งาน"
     },
     "Payload": {
-      "BoardingTime": "Boarding Time",
```

Comment:
There should not be a diff in the localisation files here, this is handled through localazy

Reply:
See below...